app.js

```diff
@@ -0,0 +1,25 @@
+require('dotenv').config();
+const cors = require('cors');
+const express = require('express');
+const mongoose = require('mongoose');
+const app = express();
+const apiRotes = require('./src/modules/routers/task-routers');
+const PORT = process.env.PORT || 5000;
```

Comment:
Move "PORT" and "URL" to "config.js" file and import these variables from there to "app" file